mfr/extensions/tabular/libs/stdlib_tools.py

```diff
@@ -30,3 +34,21 @@ def csv_stdlib(fp):
         })
     rows = [row for row in reader]
     return columns, rows
+
+
+def detect_quoted_fields(dialect, data):
```

Comment:
What are the default quoting characters? Is it possible that the user needs to escape the quotes a certain way?